src/concourse/pipelines/infrastructure/airbyte/pipeline.py

```diff
@@ -0,0 +1,71 @@
+from concourse.lib.constants import PULUMI_CODE_PATH
+from concourse.lib.jobs.infrastructure import packer_jobs, pulumi_jobs_chain
+from concourse.lib.models.fragment import PipelineFragment
+from concourse.lib.models.pipeline import GetStep, Identifier, Pipeline
+from concourse.lib.resources import git_repo, github_release
+
+airbyte_release = github_release(Identifier("airbyte-release"), "airbytehq", "airbyte")
+airbyte_image_code = git_repo(
+    Identifier("ol-infrastructure-packer"),
+    uri="https://github.com/mitodl/ol-infrastructure",
+    paths=[
+        "src/bilder/components/",
+        "src/bilder/images/airbyte/",
+    ],
+)
+
+airbyte_pulumi_code = git_repo(
+    name=Identifier("ol-infrastructure-pulumi"),
+    uri="https://github.com/mitodl/ol-infrastructure",
+    paths=[
+        "src/ol_infrastructure/applications/airbyte/",
+        "pipelines/infrastructure/scripts/",
+    ],
+)
```

Comment:
I think we are going to need to re-evaluate the `paths` that we setup as part of these packer+pulumi jobs.

I'm not sure if I can articulate, at the moment, what the issue here is but it isn't _quite_ right. 

I think the AMI paths need to be expanded to include other 'shared' things like `vector`, `vault`, `consul`, `docker-baseline-ami`, but of course this would have the side effect of trigger more builds and more builds simultaneously. 

Second, The link between the AMI bake and the pulumi state refresh + update makes sense for some situations but not others. I believe, as this pipeline is structured now, the pulumi states will be updated when a new AMI is baked but NOT when there is a change to say `src/ol-infrastructure/applications/airbyte/*` Which I think we might want, too. 

Third there is a blind break between the AMI build and the pulumi state updates. That is, there is nothing saying that the AMI made in the packer steps is what will be used in the pulumi steps. I don't think there is necesarily a resolution to this or that it is a serious issue at the moment, but I think it is something to be aware of for debugging. There really is no relationship between the two except temporal locality or something cleverly phrased like that. 

There are more problem but I can't think of them at the moment.